test/openapi3/request-oas3-test.js

```diff
@@ -226,12 +223,13 @@ describe('oas3 check', function () {
                         'bark': 'hav hav'
                     });
 
-                    const error = new InputValidationError(schemaEndpoint.body.errors,
-                        { beautifyErrors: true,
-                            firstError: true });
-                    expect(error.errors).to.be.equal('body/type should be equal to one of the allowed values [dog_object,cat_object]');
-
-                    // expect(schemaEndpoint.body.errors).to.be.equal('["Error: should be equal to one of the allowed values"]');
+                    expect(schemaEndpoint.body.errors[0].message).to.equal('should be equal to one of the allowed values');
```

Comment:
Can we make the error looks similar to the old one? (With the allowed values in the message etc)

Reply:
message can be mor clearer, but for now I dont want to change because the express-ajv-middleware use it